Create_users_on_website.py

```diff
@@ -0,0 +1,36 @@
+"""
+Example Admas created on 6/29 office hours to create random email address and add a user.
+
+Example command:
+ $ python Create_users_on_website.py --number_of_users=5
+"""
+
+from woocommerce import API
+import uuid
+import argparse
+# import sys
+# arguments = sys.argv
+# number_of_users = arguments[1]
+parser = argparse.ArgumentParser(description='Process some integers.')
+parser.add_argument('--number_of_users', required=True,
+                    help='how many users to create')
+args = parser.parse_args()
+num_of_users = args.number_of_users
+
```

Comment:
You should get the credentials from environment variables.

Reply:
Set these up and it works.